clients/src/test/java/org/apache/kafka/clients/producer/internals/RecordAccumulatorTest.java

```diff
@@ -153,7 +153,7 @@ private void verifyTopicPartitionInBatches(Map<Integer, List<ProducerBatch>> bat
             topicPartitionsInBatch.add(batchList.get(0).topicPartition);
         }
 
-        for (int i = 0 ; i < tp.length ; i++){
+        for (int i = 0; i < tp.length; i++) {
             assertEquals(tp[i], topicPartitionsInBatch.get(i));
```

Comment:
clients:checkstyleTest FAILED,  I just change the code style

Reply:
Thanks for the update. Let's wait for the jenkins build results! :)

Reply:
@showuon   jenkins build result comes out, the error seems uncorrelated, please have a check  thanks alot